src/router.js

```diff
@@ -29,11 +29,14 @@ export default new Router({
     {
       path: "/konnekt",
       name: "konnekt",
-      component: () => import("./views/Konnekt")
-    },
-    {
-      path: "/konnekt/search",
-      component: () => import("./components/KonnektSearch")
+      component: () => import("./views/Konnekt"),
```

Comment:
Change these to
```js
    {
      path: "/konnekt",
      component: Konnekt,
      children: [
        { name: "konnekt-home", path: "", component: KonnektHome },
        {
          name: "konnekt-search",
          path: "search",
          component: KonnektSearch
        }
      ]
    }
```
and use import statements at the top of the file, I think chunking all the components is not required.

Reply:
done